angular.json

```diff
@@ -37,9 +37,17 @@
             "translation": "i18n/zh-Hans/messages.zh-Hans.xlf",
             "baseHref": "/"
           },
+          "zh-SG": {
```

Comment:
Remove them as they are not supported by angular anyway and language codes will change. Like I have mentioned in https://github.com/kubernetes/dashboard/pull/4930 we will update them once angular releases patch for this. They have already fixed issue that I have created, so it should not take long.

Reply:
I need some more investigation(#4973) on Angular 9 side. Please wait.